### PR TITLE
[Refactor] Remove Cluster in Load Balancer

### PR DESCRIPTION
**Pull request by imay**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 25 / 45 (55.56%)




#### file detail

|   |path|covered line|new line|coverage|not covered line detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/proc/StatisticProcDir.java|0|1|00.00%|[149]|
|:large_blue_circle:|com/starrocks/load/ExportJob.java|0|1|00.00%|[385]|
|:large_blue_circle:|com/starrocks/common/proc/ClusterBalanceProcDir.java|0|1|00.00%|[75]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[274]|
|:large_blue_circle:|com/starrocks/common/proc/ClusterLoadStatisticProcDir.java|0|2|00.00%|[50, 51]|
|:large_blue_circle:|com/starrocks/load/loadv2/LoadLoadingTask.java|0|1|00.00%|[119]|
|:large_blue_circle:|com/starrocks/clone/BeLoadRebalancer.java|0|5|00.00%|[86, 165, 188, 189, 203]|
|:large_blue_circle:|com/starrocks/clone/TabletScheduler.java|2|8|25.00%|[341, 357, 358, 745, 905, 964]|
|:large_blue_circle:|com/starrocks/clone/ColocateTableBalancer.java|1|2|50.00%|[157]|
|:large_blue_circle:|com/starrocks/clone/DiskAndTabletLoadReBalancer.java|12|13|92.31%|[107]|
|:large_blue_circle:|com/starrocks/alter/Alter.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/alter/AlterJobV2.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/clone/Rebalancer.java|4|4|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/clone/Rebalancer.java [hunk 1]
```diff
@@ -58,10 +57,10 @@ public Rebalancer(SystemInfoService infoService, TabletInvertedIndex invertedInd
 
     public List<TabletSchedCtx> selectAlternativeTablets() {
         List<TabletSchedCtx> alternativeTablets = Lists.newArrayList();
-        for (Map.Entry<String, ClusterLoadStatistic> entry : statisticMap.entrySet()) {
+        ClusterLoadStatistic localLoadStat = loadStatistic;
+        if (localLoadStat != null) {
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/clone/Rebalancer.java:61]: You can use `loadStatistic` directly.

Comment (imay) [reply fe/fe-core/src/main/java/com/starrocks/clone/Rebalancer.java:61]: I'm afraid that loadStatisitc can be changed, so I got a reference.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 27 / 47 (57.45%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/proc/StatisticProcDir.java|0|1|00.00%|[149]|
|:large_blue_circle:|com/starrocks/load/ExportJob.java|0|1|00.00%|[385]|
|:large_blue_circle:|com/starrocks/common/proc/ClusterBalanceProcDir.java|0|1|00.00%|[75]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[274]|
|:large_blue_circle:|com/starrocks/common/proc/ClusterLoadStatisticProcDir.java|0|2|00.00%|[50, 51]|
|:large_blue_circle:|com/starrocks/load/loadv2/LoadLoadingTask.java|0|1|00.00%|[119]|
|:large_blue_circle:|com/starrocks/clone/BeLoadRebalancer.java|0|5|00.00%|[86, 165, 188, 189, 203]|
|:large_blue_circle:|com/starrocks/clone/TabletScheduler.java|4|10|40.00%|[345, 361, 362, 749, 909, 968]|
|:large_blue_circle:|com/starrocks/clone/ColocateTableBalancer.java|1|2|50.00%|[157]|
|:large_blue_circle:|com/starrocks/clone/DiskAndTabletLoadReBalancer.java|12|13|92.31%|[107]|
|:large_blue_circle:|com/starrocks/alter/Alter.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/alter/AlterJobV2.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/clone/Rebalancer.java|4|4|100.00%|[]|

**Review by wyb — APPROVED** [review]

**Pull request merged**